seldon/Makefile

```diff
@@ -7,8 +7,14 @@
 
 seldon-core-operator/base: clean-kustomize
 	mkdir -p seldon-core-operator/base
-	cd seldon-core-operator/base && helm template -f ../../values.yaml seldon-core seldon-core-operator --repo https://storage.googleapis.com/seldon-charts --namespace kubeflow --version 1.1.0  > resources.yaml
+	cd seldon-core-operator/base && helm template -f ../../values.yaml seldon-core seldon-core-operator --repo https://storage.googleapis.com/seldon-charts --namespace kubeflow --version 1.2.1  > resources.yaml
+	sed -i 's#cert-manager.io/inject-ca-from:.*#cert-manager.io/inject-ca-from: $$(CERTIFICATE_NAMESPACE)/$$(CERTIFICATE_NAME)#g' seldon-core-operator/base/resources.yaml
```

Comment:
Is this supposed to run on MacOS? If yes, please fix the `sed -i` command.

Reply:
The present OWNERS run linux. Could we punt this to a future small PR fix? @adrian555

Reply:
k